server/studyspaces.py

```diff
@@ -0,0 +1,90 @@
+from flask import request, jsonify
+from .base import *
```

Comment:
Try to avoid star imports when possible, in this case I don't think you're using anything from base